src/ResourceManager/Network/Stack/Commands.Network/Generated/Models/VirtualNetworkGatewayConnection.cs

```diff
@@ -110,13 +110,13 @@ public VirtualNetworkGatewayConnection(string etag = default(string), string aut
         /// The Egress Bytes Transferred in this connection
         /// </summary>
         [JsonProperty(PropertyName = "properties.egressBytesTransferred")]
-        public long? EgressBytesTransferred { get; set; }
+        public ulong EgressBytesTransferred { get; set; }
```

Comment:
The type change is to make the Model is in sync with PSVirtualNetworkGatewayConnection ( Automapper would threw an InvalidCastException otherwise